tensorflow/python/ops/math_ops.py

```diff
@@ -100,6 +100,7 @@
 @@complex
 @@conj
 @@imag
+@@arg
```

Comment:
We have a more extensive namespacing plan, where all the operators will eventually be in more appropriate sub-namespaces. So conj, abs, etc. will be in math. This is similar to say numpy where not everything is in the root numpy. I know in numpy we have numpy.angle, but we have already diverged on name.